brainiak/event_seg/event.py

```diff
@@ -0,0 +1,312 @@
+"""Event segmentation using a Hidden Markov Model"""
+
+# Authors: Chris Baldassano and Cătălin Iordan (Princeton University)
+
+import numpy as np
+from scipy import stats
+import logging
+
+logger = logging.getLogger(__name__)
+
+__all__ = [
+    "EventSegment",
+]
+
+
+class EventSegment:
+    """Class for learning and representing an event segmentation of
+    continuous fMRI data
```

Comment:
Please write a one-line summary (+ additional paragraphs for details, if needed):
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

We have issue #65 open for adding this check to `pr-check`.